README.md

```diff
@@ -69,3 +69,34 @@ list](https://groups.google.com/forum/#!forum/flutter-desktop-embedding-dev).
 * This is not an officially supported Google product.
 * The code and examples here, and the desktop Flutter libraries they use, are
   in early stages, and not intended for production use.
+
+# How to run Flutter desktop application in Windows
+
+    1 Switch to Master Channel
+    2 Enable Flutter Desktop
+    3 Clone the Runner Project
+    
+
+    1 How To Switch Master Channel ?
```

Comment:
This duplicates instructions linked to above. We explicitly avoid duplicating instructions, as it is a maintenance issue.